packages/nx/src/utils/output.ts

```diff
@@ -103,10 +96,10 @@ class CLIOutput {
 
   applyNxPrefix(color = 'cyan', text: string): string {
     let nxPrefix = '';
-    if (chalk[color]) {
-      nxPrefix = chalk.reset.inverse.bold[color](` ${this.cliName} `);
+    if (pc[color]) {
+      nxPrefix = pc.reset(pc.inverse(pc.bold(pc[color](` ${this.cliName} `))));
     } else {
-      nxPrefix = chalk.reset.inverse.bold.keyword(color)(` ${this.cliName} `);
```

Comment:
this used to allow any RGB colour, and now will only allow basic colours (and will otherwise just show it as bold/inverse)

I haven't been far enough down the rabbit hole to know if we ever passed a non-basic colour into this

thoughts?